src/cuga/sdk_core/debug_tool_guard_e2e.py

```
@@ -0,0 +1,385 @@
+async def create_and_process_policies(agent, policy_system):
+    """Create policies and generate examples and guard code for each."""
+    print("\nStep 1: Creating and processing policies...")
+    print("="*60)
+    
+    policy_data = []
+    
+    for idx, policy_config in enumerate(POLICIES, 1):
+        print(f"\n--- Processing Policy {idx}/{len(POLICIES)}: {policy_config['name']} ---")
+        
+        # Create policy
+        print(f"Creating policy...")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)